client/components/product-list-item.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+function ProductListItem(props) {
+  const items = props.products.map(item => {
```

Comment:
`ProductListItem` should only receive on product at a time. You should map over the list of products in `ProductList`